ui/plugins/controller.js

```diff
@@ -439,6 +439,7 @@ treeherder.controller('PluginCtrl', [
 
         var selectJobAndRender = function(job) {
             $scope.jobLoadedPromise = selectJob(job);
+            $( "#info-panel" ).addClass( "info-panel-slide" );
```

Comment:
We were doing it here https://github.com/mozilla/treeherder/blob/3e86ab64e58cb2e65803dc627d6adcbd13f5d9ad/ui/plugins/pinboard.js#L49-L51 but we can tidy that up in this PR or in a separate bug, whatever you guys prefer.